src/test/java/de/mediathekview/mserver/crawler/kika/tasks/KikaApiVideoInfoDeserializerTest.java

```diff
@@ -76,4 +44,10 @@ public void testDeserializeBrand() throws URISyntaxException, IOException {
     //
   }
 
+  static Stream<Arguments> getTestData() {
```

Comment:
Hier auch wieder. Wie wäre es mit `deserializationTestArgumentSource`

Reply:
Fixed as in [e293292](https://github.com/mediathekview/MServer/pull/925/commits/e29329208f5cffc4eda5e1a09e409ff40cb848b1)